web/client/utils/cesium/GeoJSONStyledFeatures.js

```diff
@@ -4,9 +4,13 @@ import omit from 'lodash/omit';
 import isArray from 'lodash/isArray';
 import uuid from 'uuid/v1';
 
-const validateCoordinatesValue = (coords) => {
+const validateCoordinatesValue = (coords, prevCoords) => {
```

Comment:
can you add jsdoc to the functions in this file? 

what is the structure of coords for example?